vendor/interstellar-network/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "interstellar-network",
+  "version": "0.0.12",
+  "description": "The interstellar-network module provides communication layer between Interstellar application and Stellar network.",
+  "keywords": [
+    "stellar",
+    "interstellar"
+  ],
+  "scripts": {
+    "test": "gulp test",
+    "postversion": "git push && git push --tags"
+  },
+  "author": "Stellar Development Foundation <stellar@stellar.org>",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/stellar/interstellar-network.git"
+  },
+  "dependencies": {
+    "stellar-sdk": "*",
```

Comment:
By changing this to `"*"` for the version, it's satisfied with whatever the parent package requests. This should probably have originally been a peerDep